instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/package.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-_instruments = ("sqlalchemy",)
+_instruments = ("sqlalchemy >= 1.3",)
```

Comment:
Will the instrumentation not work with <1.3 after this change? That's a deal breaker IMO.

Reply:
anything < 1.3 is marked EOL by SQLAlchemy (https://www.sqlalchemy.org/download.html). That said, I tested it with 1.2 and it worked so have removed the constraint.

Reply:
We don't have a policy that drops support for unsupported software. Companies can easily get paid support for software that'd otherwise EOL'ed for example or continue to run old software without support. Personally I think we should not drop support until it gets painful to do it for very old versions. If it works for free, let's not drop support for it.